python/chronos/src/bigdl/chronos/autots/__init__.py

```diff
@@ -24,8 +24,7 @@
                   "in your bash terminal")
 
 try:
-    # TODO: make this a LazyImport
-    from .autotsestimator import AutoTSEstimator
     from .tspipeline import TSPipeline
-except ImportError:
+    from .autotsestimator import AutoTSEstimator
+except:
```

Comment:
why we remove the `ImportError` limit here?

Reply:
undo this change.